src/rust/inetstack/protocols/udp/tests.rs

```diff
@@ -34,6 +41,8 @@ use ::std::{
     },
 };
 
+const HEADER_SIZE: usize = UDP_HEADER_SIZE + IPV4_HEADER_MAX_SIZE as usize + ETHERNET2_HEADER_SIZE;
```

Comment:
Should this be made a constant in mailine code?

Reply:
I switched it to MAX_HEADER_SIZE